BiasAndFairnessServers/src/crud/bias_and_fairness.py

```
@@ -0,0 +1,43 @@
+                d.id AS data_id,
+                f.id AS metrics_id,
+                f.metrics AS metrics
+            FROM fairness_runs f JOIN model_data d ON f.data_id = d.id WHERE f.id = :id;"""),
+        {"id": id}
+    )
+    row = result.mappings().first()
+    return row
+
+async def upload_model(content, name, db):
+    result = db.execute(
+        text("INSERT INTO model_files (name, file_content) VALUES (:name, :file_content) RETURNING id"),
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)